packages/libs/react-ui/src/components/NavHeader/NavHeader.stories.tsx

```diff
@@ -88,21 +97,28 @@ export const Dynamic: IStory = {
   args: {
     brand: logoVariants[0],
     linksCount: 3,
+    navHeaderActiveLink: 0,
     customNavigation: sampleNavItems,
   },
   render: ({
     brand,
     useCustomNavigation,
     customNavigation,
     linksCount,
+    navHeaderActiveLink,
     renderSampleContent = false,
   }) => {
     const navItems = useCustomNavigation ? customNavigation : sampleNavItems;
+
     return (
       <NavHeader.Root brand={brand}>
-        <NavHeader.Navigation>
+        <NavHeader.Navigation activeLink={navHeaderActiveLink}>
           {navItems.slice(0, linksCount).map((item, index) => (
-            <NavHeader.Link key={index} href={item.href}>
+            <NavHeader.Link
+              key={index}
+              href={item.href}
+              onClick={(event) => console.log(item.label, { event })}
```

Comment:
Should we treat this link like our other Link component and accept all link related props that don't affect the props we are using?